src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -23,7 +23,7 @@ public partial class FakeManager : IFakeCallProcessor
             new DefaultReturnValueRule(),
         };
 
-        private readonly IFakeObjectCallRule[] preUserRules;

```

Comment:
It was the only pre-user rule, so I removed the field. We can always recreate it if we need to add a new pre-user rule.